src/main/java/jenkins/plugins/http_request/auth/FormAuthentication.java

```diff
@@ -71,5 +71,10 @@ public FormValidation doCheckKeyName(@QueryParameter String value) {
         public String getDisplayName() {
             return "Form Authentication";
         }
+
+        public String getHttpRequestHelpPath() {
+            return "/descriptor/"+HttpRequest.class.getName()+"/help";
```

Comment:
Why did you add this? Help paths are automatically managed by Jenkins core (based on naming conventions). Are you trying to achieve some special behaviour?